packages/react/src/button/Button.tsx

```diff
@@ -40,6 +43,7 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
 		{
 			block = false,
 			children,
+			focusRingFor,
```

Comment:
Perhaps it would be better to instead have `'keyboard'` as the default here so people can see that in the Code tab etc

Reply:
I'll also add it to the `ButtonLink`. I didn't see it lurking down there in the same file. FWIW I prefer to have files export only one component as things tend to get lost and/or unwieldy.